Missing CLog.cmake file

I am following the setup steps according to your documentation (https://github.com/microsoft/msquic/blob/main/docs/BUILD.md). 
I reached the configuration step where I have to run:
`./scripts/prepare-machine.ps1 -Configuration Dev`

However, this script returns the following output:
-------------------



PS /home/simonas_client/Documents/msquic> ./scripts/build.ps1
[11/15/2020 20:51:06] Generating files...
[11/15/2020 20:51:06] cmake -g 'Ninja' -DQUIC_TLS=openssl -DQUIC_OUTPUT_DIR=/home/simonas_client/Documents/msquic/artifacts/bin/linux/x64_Debug_openssl -DQUIC_ENABLE_LOGGING=on -DCMAKE_BUILD_TYPE=Debug ../../..
-- CMAKE Version: 3.16.3
-- Source Dir: /home/simonas_client/Documents/msquic
-- Host System name: Linux
-- Setting policy 0091
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is Clang 10.0.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- System name: Linux
-- System version: 5.4.0-52-generic
-- Platform version: 
-- Build type: Debug
-- QUIC Platform: linux
-- Version Build ID: 0
-- Version Suffix: -private
CMake Error at CMakeLists.txt:187 (include):
  include could not find load file:

    /home/simonas_client/Documents/msquic/build/linux/x64_openssl/clog/CLog.cmake


-- Configuring for LTTng tracing
-- Found LTTngUST: /usr/lib/x86_64-linux-gnu/liblttng-ust.so;dl (found version "2.1.0") 
-- Disabling session resumption support
-- Disabling 0-RTT support
-- Found libatomic: /usr/lib/x86_64-linux-gnu/libatomic.so.1
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
CMake Error at CMakeLists.txt:190 (CLOG_GENERATE_TARGET):
  Unknown CMake command "CLOG_GENERATE_TARGET".
Call Stack (most recent call first):
  src/core/CMakeLists.txt:47 (add_clog_library)


-- Configuring incomplete, errors occurred!
See also "/home/simonas_client/Documents/msquic/build/linux/x64_openssl/CMakeFiles/CMakeOutput.log".
See also "/home/simonas_client/Documents/msquic/build/linux/x64_openssl/CMakeFiles/CMakeError.log".
Write-Error: /home/simonas_client/Documents/msquic/scripts/build.ps1:310
Line |
 310 |      CMake-Execute $Arguments
     |      ~~~~~~~~~~~~~~~~~~~~~~~~
     | [11/15/2020 20:51:07] CMake exited with status code 1










-------------------

Do you know where the issue is, because CMakeOutput.log and CMakeError.log aren't very informative (but I can add them, if that would help).
I am running Ubuntu20.04, my cmake version is 3.16.3 and I am using Powershell in the terminal.

--- a/docs/BUILD.md
+++ b/docs/BUILD.md
@@ -1,6 +1,6 @@
 # Building MsQuic
 
-The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK or better) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows) (7.0 or better) on all platforms.
+The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows) (7.0 or better) on all platforms.
 
 > **Note** - clone the repo recursively or run `git submodule update --init --recursive`
 to get all the submodules.

--- a/docs/BUILD.md
+++ b/docs/BUILD.md
@@ -1,6 +1,6 @@
 # Building MsQuic
 
-The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows) (7.0 or better) on all platforms.
+The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) (7.0 or better) on all platforms.
 
 > **Note** - clone the repo recursively or run `git submodule update --init --recursive`
 to get all the submodules.

--- a/docs/BUILD.md
+++ b/docs/BUILD.md
@@ -1,6 +1,6 @@
 # Building MsQuic
 
-The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) (7.0 or better) on all platforms.
+The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 or better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) (7.0 or better) on all platforms.
 
 > **Note** - clone the repo recursively or run `git submodule update --init --recursive`
 to get all the submodules.

--- a/docs/BUILD.md
+++ b/docs/BUILD.md
@@ -1,6 +1,6 @@
 # Building MsQuic
 
-MsQuic uses [CMake](https://cmake.org/) to generate build files.
+The full MsQuic build system relies on [CMake](https://cmake.org/) (3.16 of better), [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) (3.1 SDK or better) and [Powershell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows) (7.0 or better) on all platforms.
 
 > **Note** - clone the repo recursively or run `git submodule update --init --recursive`
 to get all the submodules.

